examples/cassandra/README.md

```diff
@@ -300,47 +226,106 @@ spec:
 [Download example](cassandra-controller.yaml?raw=true)
 <!-- END MUNGE: EXAMPLE cassandra-controller.yaml -->
 
-Most of this replication controller definition is identical to the Cassandra pod
-definition above; it simply gives the replication controller a recipe to use
-when it creates new Cassandra pods.  The other differentiating parts are the
-`selector` attribute which contains the controller's selector query, and the
-`replicas` attribute which specifies the desired number of replicas, in this
-case 1.
+There are a few things to note in this description.
+
+The `selector` attribute contains the controller's selector query. It can be
```

Comment:
Same as with the comment above, see https://github.com/kubernetes/kubernetes/issues/2210 , and a discussion of where you might want to use manual selectors.
But as to the .yaml comment on the selector label -- suspect this may in part have been added as fyi for longer-term Kubernetes users, since this is new-ish.